src/main/java/uk/gov/ons/ctp/integration/rhsvc/representation/WebformDTO.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.ons.ctp.integration.rhsvc.representation;
+
+import java.io.Serializable;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.common.domain.Region;
+
+/** This object holds details from a webform. */
+@Data
+@NoArgsConstructor
+@SuppressWarnings("serial")
+public class WebformDTO implements Serializable {
+
+  /** enum for category */
+  public enum WebformCategory {
+    MISSING_INFORMATION,
+    TECHNICAL,
+    FORM,
+    COMPLAINT,
+    ADDRESS,
+    OTHER
+  }
+
+  /** enum for language */
+  public enum WebformLanguage {
+    EN,
+    CY
+  }
+
+  @NotNull private WebformCategory category;
+
+  @NotNull private Region region;
+
+  @NotNull private WebformLanguage language;
+
+  @NotNull private String name;
```

Comment:
JIRA suggests some constraints on name, description, email.  Please check if we need additional validation here.

Reply:
Have discussed this with Ameet and Paul. RHSvc not enforcing as done by RHUI.
Jira ticket updated with note about this.